### DISCUSSION TITLE
Drapzone，拖动的时候，显示的内容和选择的时候选择的内容不一致。

### DISCUSSION THREAD

**Discussion by supperlitt** [dotnetcore/BootstrapBlazor#4229] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

我选择的，Test1，然后拖动过程中，名称变成了Test2或者他。

[BlazorApp1.zip](https://github.com/user-attachments/files/16904341/BlazorApp1.zip)

**Comment by bb-auto** [comment]
@supperlitt Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@supperlitt 感谢大佬提交 Issue 推荐使用 `SortableList` 组件 文档如下：https://www.blazor.zone/sortable-list

**Comment by supperlitt** [comment]
大佬，sortlist试了，感觉是直接拷贝的元素内容，但是我虽然是拖动，但是我要进行二次渲染，内容是不一样的。

**Comment by ArgoZhang** [comment]
@supperlitt 那个页面？如何操作？没有操作步骤与截图。`DrapZone` 准备移除掉，全面使用 `SortableList` 组件，如果有什么需求可以提

**Reply by ArgoZhang** [reply]
提交人后续无跟进，转移为讨论
